source/command/atlas-kubernetes-config-generate.txt

```diff
@@ -53,10 +53,14 @@ Options
      - 
      - false
      - Flag that generates kubernetes secrets with data for projects, users, deployments entities.
+   * - --independentResources
+     - 
+     - false
+     - Flag that makes the generated resources that support independent usage, to use external IDs rather than Kubernetes references.
```

Comment:
```suggestion
     - Flag that indicates whether the generated resources that support independent usage use external IDs rather than Kubernetes references.
```